js/apps/admin-ui/src/identity-providers/add/DetailSettings.tsx

```diff
@@ -101,6 +109,44 @@ const Header = ({ onChange, value, save, toggleDeleteDialog }: HeaderProps) => {
     },
   });
 
+  const importSamlKeys = async (
+    providerId: string,
+    entityDescriptorUrl: string,
+  ) => {
+    try {
+      const result = await adminClient.identityProviders.importFromUrl({
+        providerId: providerId,
+        fromUrl: entityDescriptorUrl,
+      });
+      if (result["signingCertificate"]) {
+        setValue(`config.signingCertificate`, result["signingCertificate"]);
```

Comment:
Unless the key is special and needs escaping we prefer the dot notation.
```suggestion
      if (result.signingCertificate) {
        setValue(`config.signingCertificate`, result.signingCertificate);
```

Reply:
Done!